maskfile.md

```diff
@@ -27,6 +27,26 @@ else
 fi
 ~~~
 
+~~~powershell
```

Comment:
Since this is the root maskfile and people may be reading this, adding a comment here to explain why this works would be nice I think.

```suggestion
**Note:** On Windows platforms, `mask` falls back to running `powershell` code blocks.

~~~powershell
```